### PR TITLE
Ins 2828 get jettree from heavy

### PR DESCRIPTION
**Pull request by ivanshibitov**
**- What I did**
1. Go to heavy from JetFetcher
2. handler GetJet message on heavy

**- How I did it**

**- How to verify it**

**- Description for the changelog**

### CODE REVIEW

File: insolar/jet/fetcher.go [hunk 1]
```diff
@@ -301,9 +301,17 @@ func (tu *fetcher) nodesForPulse(ctx context.Context, pulse insolar.PulseNumber)
 
 	res, err := tu.Nodes.InRole(pulse, insolar.StaticRoleLightMaterial)
 	if err != nil {
-		return nil, err
+		return nil, errors.Wrap(err, "[ nodesForPulse ] Can't get node of 'light' role")
```
Comment (andreyromancev) [inline insolar/jet/fetcher.go]: We don't use special formatting in error text.

File: ledger/heavy/handler/handler_test.go [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+		actual bool
+	}{
+		{name: "Actual", actual: true},
+		{name: "NotActual", actual: false},
+	}
+	for _, tt := range ttable {
+		t.Run(tt.name, func(t *testing.T) {
+			testCtx := inslogger.TestContext(t)
+			testPulseNumber := insolar.PulseNumber(998999)
+			testID := testutils.RandomID()
```
Comment (andreyromancev) [inline ledger/heavy/handler/handler_test.go]: `gen.ID()`

File: ledger/heavy/handler/handler_test.go [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+		name   string
+		actual bool
+	}{
+		{name: "Actual", actual: true},
+		{name: "NotActual", actual: false},
+	}
+	for _, tt := range ttable {
+		t.Run(tt.name, func(t *testing.T) {
+			testCtx := inslogger.TestContext(t)
+			testPulseNumber := insolar.PulseNumber(998999)
```
Comment (andreyromancev) [inline ledger/heavy/handler/handler_test.go]: `testPulseNumber := insolar.FirstPulseNumber + N`

File: insolar/jet/fetcher.go [hunk 4]
```diff
@@ -301,9 +301,17 @@ func (tu *fetcher) nodesForPulse(ctx context.Context, pulse insolar.PulseNumber)
 
 	res, err := tu.Nodes.InRole(pulse, insolar.StaticRoleLightMaterial)
```
Comment (andreyromancev) [inline insolar/jet/fetcher.go]: This will return `node.ErrNoNodes`. We need to catch it and **only then** return heavy node for **current pulse**.

Comment (andreyromancev) [reply insolar/jet/fetcher.go]: Also, please write a test for this case.

Comment (ivanshibitov) [reply insolar/jet/fetcher.go]: I think this logic is not correct. When we have just started on existing db, we will have Light nodes, but all of them might not have actual jets.
About tests - yes, sure!

**Pull request merged**